sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -748,6 +748,13 @@ class SessionCatalog(
     getRawLocalOrGlobalTempView(toNameParts(name)).map(getTempViewPlan)
   }
 
+  /**
```

Comment:
An overloaded version that takes name parts directly instead of taking `TableIdentifier`.